scipy/stats/_pagel.py

```diff
@@ -0,0 +1,287 @@
+from collections import namedtuple
+import numpy as np
+from ._continuous_distns import chi2, norm
+import scipy.stats
+
+
+Page_L_Result = namedtuple('Page_L_Result',
+                           ('statistic', 'pvalue', 'method'))
+
+
+def pagel(data, ranked=True, predicted_ranks=None, method='auto', n_s=2000,
+          ties=None):
+    r"""
+    Compute Page's L, a measure of trend in observations between treatments.
+
+    Page's L is useful when:
+
+        * there are :math:`n \geq 3` treatments,
+        * :math:`m \geq 2` subjects are observed for each treatment, and
+        * the observations are hypothesized to have a particular order.
+
```

Comment:
The [Wikipedia page](https://en.wikipedia.org/wiki/Page%27s_trend_test) has useful context here, like it's related to `spearmanr` and has more statistical power than `friedmanchisquare`. I think when we add more of these lesser-known tests, such context is becoming more and more important to add.

Reply:
OK, adding.